app/src/main/res/layout/dialog_playback_parameter.xml

```diff
@@ -261,11 +261,121 @@
         </RelativeLayout>
 
+        <RelativeLayout
+            android:id="@+id/semitoneControl"
+            android:layout_width="match_parent"
+            android:layout_height="40dp"
+            android:layout_below="@id/pitchControlText"
+            android:layout_marginTop="4dp"
+            android:orientation="horizontal">
+
+            <org.schabi.newpipe.views.NewPipeTextView
+                android:id="@+id/semitoneStepDown"
+                android:layout_width="24dp"
+                android:layout_height="match_parent"
+                android:layout_alignParentStart="true"
+                android:layout_alignParentLeft="true"
+                android:layout_centerVertical="true"
+                android:background="?attr/selectableItemBackground"
+                android:clickable="true"
+                android:focusable="true"
+                android:gravity="center"
+                android:text="♭"
+                android:textColor="?attr/colorAccent"
+                android:textSize="24sp"
+                android:textStyle="bold"
+                tools:ignore="HardcodedText"
+                tools:text="♭" />
+
+            <RelativeLayout
+                android:id="@+id/semitoneDisplay"
+                android:layout_width="match_parent"
+                android:layout_height="match_parent"
+                android:layout_marginLeft="4dp"
+                android:layout_marginRight="4dp"
+                android:layout_toStartOf="@+id/semitoneStepUp"
+                android:layout_toLeftOf="@+id/semitoneStepUp"
+                android:layout_toEndOf="@+id/semitoneStepDown"
+                android:layout_toRightOf="@+id/semitoneStepDown"
+                android:orientation="horizontal">
+
+                <org.schabi.newpipe.views.NewPipeTextView
+                    android:id="@+id/semitoneMinimumText"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:layout_alignParentStart="true"
+                    android:layout_alignParentLeft="true"
+                    android:layout_marginStart="4dp"
+                    android:layout_marginLeft="4dp"
+                    android:gravity="center"
+                    android:text="-12"
+                    android:textColor="?attr/colorAccent"
+                    tools:ignore="HardcodedText"
+                    tools:text="-12" />
+
+                <org.schabi.newpipe.views.NewPipeTextView
+                    android:id="@+id/semitoneCurrentText"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:layout_centerHorizontal="true"
+                    android:gravity="center"
+                    android:text="--"
+                    android:textColor="?attr/colorAccent"
+                    android:textStyle="bold"
+                    tools:ignore="HardcodedText"
```

Comment:
And here no need for hardcoded text, since the value is always set dynamically
```suggestion
                    android:textColor="?attr/colorAccent"
                    android:textStyle="bold"
```